code/datums/gamemode/dynamic/dynamic_rulesets.dm

```diff
@@ -237,6 +237,13 @@
 
 /datum/dynamic_ruleset/roundstart/ready(var/forced = 0)
 	if (!forced)
-		if(!check_enemy_jobs())
+		var/job_check = 0
```

Comment:
Can you override check_enemy_jobs instead? It's the whole point.

Reply:
fine